modules/Microsoft.Network/networkManagers/.test/common/deploy.test.bicep

```diff
@@ -0,0 +1,203 @@
+    virtualNetworkSpoke1Name: 'dep-<<namePrefix>>-vnetSpoke1-${serviceShort}'
+    virtualNetworkSpoke2Name: 'dep-<<namePrefix>>-vnetSpoke2-${serviceShort}'
+    location: location
+  }
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../deploy.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    enableDefaultTelemetry: enableDefaultTelemetry
+    name: '<<namePrefix>>${serviceShort}001'
+    lock: 'CanNotDelete'
+    roleAssignments: [
+      {
+        roleDefinitionIdOrName: 'Reader'
+        principalIds: [
+          resourceGroupResources.outputs.managedIdentityPrincipalId
+        ]
+        principalType: 'ServicePrincipal'
+      }
+    ]
+    networkManagerScopeAccesses: [
+      'Connectivity'
+      'SecurityAdmin'
+    ]
+    networkManagerScopes: {
+      subscriptions: [
+        subscription().id
+      ]
+    }
+    networkGroups: [
+      {
+        name: 'network-group-spokes'
+        description: 'network-group-spokes description'
+        staticMembers: [
+          {
+            name: 'virtualNetworkSpoke1'
+            resourceId: resourceGroupResources.outputs.virtualNetworkSpoke1Id
+          }
+          {
+            name: 'virtualNetworkSpoke2'
+            resourceId: resourceGroupResources.outputs.virtualNetworkSpoke2Id
+          }
+        ]
+      }
+    ]
+    connectivityConfigurations: [
+      {
+        name: 'hubSpokeConnectivity'
+        description: 'hubSpokeConnectivity description'
+        connectivityTopology: 'HubAndSpoke'
+        hubs: [
+          {
+            resourceId: resourceGroupResources.outputs.virtualNetworkHubId
+            resourceType: 'Microsoft.Network/virtualNetworks'
+          }
+        ]
+        deleteExistingPeering: 'True'
+        isGlobal: 'True'
+        appliesToGroups: [
+          {
+            networkGroupId: concat(resourceGroup.id, '/providers/Microsoft.Network/networkManagers/,<<namePrefix>>${serviceShort}001,/networkGroups/network-group-spokes')
+            useHubGateway: 'False'
+            groupConnectivity: 'None'
+            isGlobal: 'False'
+          }
+        ]
+      }
+      {
+        name: 'MeshConnectivity'
+        description: 'MeshConnectivity description'
+        connectivityTopology: 'Mesh'
+        deleteExistingPeering: 'True'
+        isGlobal: 'True'
+        appliesToGroups: [
+          {
+            networkGroupId: concat(resourceGroup.id, '/providers/Microsoft.Network/networkManagers/,<<namePrefix>>${serviceShort}001,/networkGroups/network-group-spokes')
+            useHubGateway: 'False'
+            groupConnectivity: 'None'
+            isGlobal: 'False'
+          }
+        ]
+      }
+    ]
+    scopeConnections: [
+      {
+        name: 'scope-connection-test'
+        description: 'description of the scope connection'
+        resourceId: '/subscriptions/<<subscriptionId>>'
```

Comment:
Should work via the `subscription().id` function

Reply:
I left this on purpose to demonstrate that it can be multiple values. I'll update it and provide details in a parameter usage section.

Reply:
@AlexanderSehr Updated